tools/create_operators_pull_request.py

```diff
@@ -0,0 +1,323 @@
+    )
+    if rc != 0:
+        return rc, error_message, "", ""
+
+    return 0, "", operators_directory, operator_directory
+
+
+def commit_copied_operator_files_and_push_branch(
+    operators_directory: str,
+    operators_repository: str,
+    operators_branch: str,
+    operator_directory: str,
+    operator_name: str,
+    operator_git_tag: str,
+    git_commit_message: bool,
+    debug: bool,
+):
+    """Copies, commits and pushes operator-related files from the operator
+    repository (e.g., kudo-cassandra-operator) directory into the "operators
+    collection" repository (e.g., kudobuilder/operators) directory."""
+
+    versioned_operator_directory = (
+        f"{operators_directory}/repository/{operator_name}/{operator_git_tag}"
+    )
+
+    command = " && ".join(
+        [
+            f"mkdir -p {versioned_operator_directory}",
+            f"cp -r {operator_directory}/operator {versioned_operator_directory}",
+            f"cp -r {operator_directory}/docs {versioned_operator_directory}",
+        ]
+    )
+
+    rc, stdout, stderr = run(command, debug=debug)
+    if rc != 0:
+        return rc, f"stdout:\n{stdout}\nstderr:\n{stderr}"
+
+    rc, stdout, stderr = run(f"git -C {operators_directory} add .", debug=debug)
+    if rc != 0:
+        return rc, f"stdout:\n{stdout}\nstderr:\n{stderr}"
+
+    rc, stdout, stderr = run(
+        f"git -C {operators_directory} commit -am '{git_commit_message}'",
+        debug=debug,
+    )
+    if rc != 0:
+        return rc, f"stdout:\n{stdout}\nstderr:\n{stderr}"
+
+    rc, stdout, stderr = run(
+        f"git -C {operators_directory} log -n 1", debug=debug
+    )
+    if rc != 0:
+        return rc, f"stdout:\n{stdout}\nstderr:\n{stderr}"
+
+    rc, stdout, stderr = run(
+        f"git -C {operators_directory} push origin {operators_branch}",
+        debug=debug,
+    )
+    if rc != 0:
+        return (
+            rc,
+            f"Failed to push '{operators_branch}' to '{operators_repository}'"
+            + f"\nstdout:\n{stdout}\nstderr:\n{stderr}",
+        )
+
+
+def automated_operators_repository_commit_message(
+    operator_name: str, operator_git_tag: str
+) -> str:
+    return f"Release {operator_name} {operator_git_tag} (automated commit)."
+
+
+def automated_operators_repository_branch(
+    operator_name: str, operator_git_tag: str
+) -> str:
+    return f"{operator_name}_{operator_git_tag}_{random_short_string()}"
+
+
+def main() -> int:
+    args = parse_arguments()
+
+    operator_repository = args.operator_repository
+    operator_name = args.operator_name
+    operator_git_tag = args.operator_git_tag
+    github_token = args.github_token
+    git_commit_message = (
+        args.git_commit_message
+        or automated_operators_repository_commit_message(
+            operator_name, operator_git_tag
+        )
+    )
+    operators_base_branch = args.operators_base_branch
+    operators_repository = args.operators_repository or OPERATORS_REPOSITORY
```

Comment:
Why not rely on the default?

Reply:
Thanks! d4dc22b987ab52568e9ffae0731b8c0bb39433ab